bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/FireballListener.java

```diff
@@ -32,12 +40,22 @@ public class FireballListener implements Listener {
     private final double damageSelf;
     private final double damageEnemy;
     private final double damageTeammates;
+    private final boolean fireballDestroyBlocks;
+    private final double fireballBlockFailureRadius;
+    private final List<String> fireballBlocksThatAllowDestruction;
+    private final boolean fireballDestroyOriginalBlocks;
```

Comment:
That should be done trough nms. Just add a new function in VersionSupport for this particle so you can implement support for all mc versions supported by bedwars1058.